R/readData.R

```diff
@@ -73,21 +145,9 @@ readTableFast<-function(filename,header=TRUE,skip=0,sep=""){
 #' @rdname readGeneric
 readGeneric<-function(file, chr=1,start=2,end=3,strand=NULL,meta.cols=NULL, 
                       keep.all.metadata=FALSE, zero.based=FALSE, 
-                      remove.unusual=FALSE, header=FALSE, 
-                      skip=0, sep="\t"){
+                      remove.unusual=FALSE, header="auto", 
+                      skip=0, sep="auto"){
```

Comment:
why is header and sep "auto' ? If you are changing these you have to change the documentation (function help pages ) as well. My suggestion is to leave them as they are. The "auto" option is not compatible with current help pages or people who are used to base::read.table